bsmetadata/utils.py

```diff
@@ -0,0 +1,20 @@
+import importlib.util
```

Comment:
I don't fully understand why we need all of this logic. Wouldn't it make more sense to just directly add `datasets` to the project requirements and replace

```python
if _datasets_available:
    from datasets import Value
```
with `from datasets import Value`?

Reply:
I just thought that if we wanted to use this part without datasets it wouldn't block (because in the end datasets is not the only lib available). But if it confuses you, I can remove it. :blush:

Reply:
`datasets` is already in the `requirements.txt`

Reply:
@timoschick , as for me this feature is not at all as important as the rest, I removed it in my last commit :slightly_smiling_face: